TrafficCrash/Pages/Index.cshtml.cs

```diff
@@ -24,28 +24,23 @@ public IndexModel(ILogger<IndexModel> logger)
 
         public JsonResult OnGet()
         {
-            List<Vehicle_People_Crash_details> vehicle_n_people = new List<Vehicle_People_Crash_details>();
+            List<VehiclePeopleCrashdetails> vehicle_n_people = new List<VehiclePeopleCrashdetails>();
             using (WebClient webClient = new WebClient())
             {
                 string vehicleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/68nd-jvt3.json");
                 TrafficCrashVehicle[] allVehicles = TrafficCrashVehicle.FromJson(vehicleJson);
 
                 string peopleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/u6pd-qa9d.json");
                 TrafficCrashPeople[] allPeople = TrafficCrashPeople.FromJson(peopleJson);
-             
-
-                // this new array will hold only specimens that like water.
```

Comment:
Good to remove commented-out code.  It's considered bad practice to push commented-out code to VCS, because you can always go back in history to see what used to be there.